tests/store/test_sqlalchemy_store.py

```diff
@@ -37,6 +34,14 @@ def _experiment_factory(self, names):
 
         return self.store.create_experiment(name=names)
 
+    def test_default_experiment(self):
```

Comment:
Would it be difficult to test that I can delete the default experiment and have it not come back, assuming that I created at least one other experiment?

Reply:
yes! Was a bit of a pain but wrote a testcase that pins a sqlite file for a test. in the middle of the test, it destroys `SqlAlchemyStore`, and ensures that deleted `Default` experiment stays deleted with new `SqlAlchemyStore` using same sqlite file.

Thanks for the idea...